src/scripts/main.js

```diff
@@ -1,3 +1,48 @@
 'use strict';
 
-// write code here
+const fullList = document.querySelector('.tree');
+const mainArrLi = [...fullList.children].map((li) => li);
+const subArrLi = [...fullList.children]
```

Comment:
Consider making this logic more flexible and not depend on just 2 levels of nesting

Reply:
logic is changed